pom.xml

```diff
@@ -1360,6 +1360,13 @@
                   </bannedImports>
                   <includeTestCode>true</includeTestCode>
                 </restrictImports>
+                <restrictImports implementation="de.skuzzle.enforcer.restrictimports.RestrictImports">
+                  <reason>Do not use commons-logging; use slf4j</reason>
+                  <bannedImports>
+                    <bannedImport>org.apache.commons.logging.**</bannedImport>
```

Comment:
what do you think about `org.apache.log4j`? is that okay to use?

Reply:
I'm not sure. We have 4 imports altogether; but these were present in HIVE-20019 too.

```
accumulo-handler/src/java/.../accumulo/mr/HiveAccumuloTableInputFormat.java:import org.apache.log4j.Level;
accumulo-handler/src/test/.../accumulo/TestHiveAccumuloHelper.java:import org.apache.log4j.Logger;
llap-server/src/java/.../llap/daemon/impl/ContainerRunnerImpl.java:import org.apache.log4j.NDC;
llap-server/src/java/.../llap/daemon/impl/TaskRunnerCallable.java:import org.apache.log4j.NDC;

```

Reply:
okay; removing commons-logging usage is usefull initself ; and that NDC usage is interesting - might worth open a separate ticket to investigate that